internal/action/otp.go

```diff
@@ -108,18 +105,30 @@ func (s *Action) otp(ctx context.Context, name, qrf string, clip, pw, recurse bo
 			return nil
 		default:
 		}
-		two, label, err := otp.Calculate(name, sec)
+
+		two, err := otp.Calculate(name, sec)
```

Comment:
`two` supports both HOTP and TOTP as per the doc of `pquerna/otp` if it took the happy path where the URL is parsed easily in `otp.Calculate`.